### PR TITLE
GDB-6865 Add legend to cluster management view

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- added a legend with all node and link states
 - added button to toggle legend visibility

### CODE REVIEW

File: src/js/angular/clustermanagement/directives.js [hunk 1]
```diff
@@ -155,3 +274,26 @@ clusterManagementDirectives.directive('clusterGraphicalView', ['$window', 'Local
         };
     }
 ]);
+
+const getLegendNodes = function () {
+    const legendNodes = [];
+    legendNodes.push({nodeState: NodeState.LEADER, customText: 'Leader'});
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/directives.js:280]: i18n

File: src/pages/cluster-management/clusterInfo.html [hunk 2]
```diff
@@ -72,6 +72,14 @@ <h1>
 
 <div ng-if="!loader" class="nodes">
     <div cluster-graphical-view></div>
+    <div id="toggle-legend-button" class="toggle-legend-btn">
+        <button class="btn btn-link"
+                ng-click="toggleLegend()"
+                tooltip="{{'cluster_management.cluster_page.toggle_legend_btn' | translate}}"
+                tooltip-placement="right">
+            <span class="icon-help legend-toggle-icon"></span>
```
Comment (svilenvelikov) [inline src/pages/cluster-management/clusterInfo.html:80]: legend-toggle-icon -> toggle-legend-icon in order to be consistent with the button class

**Review by svilenvelikov — DISMISSED** [review]

File: src/js/angular/clustermanagement/directives.js (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/directives.js:280]: I will add i18n for the whole cluster drawing in a following PR

File: src/pages/cluster-management/clusterInfo.html (hunk 2 above)
Comment (yordanalexandrov) [reply src/pages/cluster-management/clusterInfo.html]: changed

File: src/pages/cluster-management/clusterInfo.html [hunk 3]
```diff
@@ -77,7 +77,7 @@ <h1>
                 ng-click="toggleLegend()"
                 tooltip="{{'cluster_management.cluster_page.toggle_legend_btn' | translate}}"
                 tooltip-placement="right">
-            <span class="icon-help legend-toggle-icon"></span>
+            <span class="icon-help toggle-legend--icon"></span>
```
Comment (svilenvelikov) [inline src/pages/cluster-management/clusterInfo.html:80]: Is this double hyphen intended or it's hust a typo or you invented some sort of BEM css selector style here?!

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**